ops/cli.py

```diff
@@ -265,7 +294,17 @@ def deploy(self, arguments) -> int:
             shutil.copy(f"gotc-deploy/deploy/gotc-dev/helm/{values}.ctmpl", dir)
             render_ctmpl(ctmpl_file=f"{values}.ctmpl", WFL_VERSION=tag)
             helm_deploy_wfl(values=values)
+
+            with open(values) as f:
+                helm_values = yaml.safe_load(f)
```

Comment:
is `help_values` used anywhere?

Reply:
Yeah! `helm_values` is used on the next two lines to get the postgres username and password.

Reply:
Ah sorry, the formatting threw me a bit...